### PR TITLE
Short version, performance increased tested in perf.js

### PR DESCRIPTION
**Pull request by bowen31337**
run perf.js in perf folder

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,19 +1,6 @@
 'use strict';
 module.exports = leftPad;
 
-var cache = [
```
Comment (stevemao) [inline index.js:4]: This is for common use-cases.

**Comment by bowen31337** [comment]
The cached for common user-cases was added back.

**Comment by stevemao** [comment]
The changes here doesn't seem to be necessary at all

**Comment by stevemao** [comment]
@bowen31337 ping.

**Pull request closed without merging**